### ISSUE TITLE
Move Sketchbook.js to it's own team and setup publishing process

### ISSUE THREAD

**Issue by haydn** [sketchbook-js/sketchbook#100]
# Objective

Set up GitHub and npm so that team members publish new releases. We'll use [np](https://github.com/sindresorhus/np) to publish releases.

# Plan

- [x] Create a GitHub organisation for Sketchbook.js (`sketchbook-js`). @haydn
- [x] Create an npm organisation for Sketchbook.js (`@sketchbook-js`). @haydn
- [x] Move the GitHub repo from `haydn/sketchbook-js` to `sketchbook-js/sketchbook`. @haydn
- [x] Set up a Vercel team a move the automatic deployments across to it. @haydn
- [x] Set up Travis CI for the new repo. @haydn
- [x] Add `np` as a dev dependency. @Mxchaeltrxn
- [x] Rename the package in `package.json` to `@sketchbook-js/sketchbook` to use the [npm organisation's scope](https://docs.npmjs.com/creating-and-publishing-an-organization-scoped-package). @Mxchaeltrxn
- [x] Reset the version in `package.json` to `0.0.0` as [recommended in the np docs](https://github.com/sindresorhus/np#initial-version). @Mxchaeltrxn
- [x] Add a `release` script as [described in the np docs](https://github.com/sindresorhus/np#release-script). ([example](https://github.com/haydn/use-state-snapshots/blob/master/package.json#L21)) @Mxchaeltrxn
- [x] Configure the [files option](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files) in `package.json` to include the files we want to distribute. For Sketchbook this'll be everything in `build` and `cli` folders + `README.md` and `LICENSE`. ([example](https://github.com/haydn/use-state-snapshots/blob/master/package.json#L23-L29)) @Mxchaeltrxn
- [x] Update `README.md` to use a build status badge for the new repo (`sketchbook-js/sketchbook`) and to link to the new deployment URL (`https://sketchbook-js.vercel.app/`). @Mxchaeltrxn
- [x] Update `CONTRIBUTING.md` to use the new repo (`sketchbook-js/sketchbook`). @Mxchaeltrxn
- [x] Publish a "preminor" release to npm (should publish version `0.1.0-0`) with a `dev` [dist-tag](https://docs.npmjs.com/cli/v6/commands/npm-dist-tag) using `yarn release --tag=dev`. @Mxchaeltrxn
- [x] Update https://sketchbook-js.particlesystem.com/ references to https://sketchbook-js.vercel.app/ @Mxchaeltrxn 

We should then be able to test everything is working by installing the prerelease version into a project:

```
yarn add @sketchbook-js/sketchbook@dev
```

**Comment by haydn** [comment]
@Mxchaeltrxn Alrighty! I've added a plan to the description here. Let me know if you've got any questions. 😄

**Comment by haydn** [comment]
@Mxchaeltrxn We've moved! You should have access to everything, shout out if there's anything you don't have permissions for.

**Comment by haydn** [comment]
@Mxchaeltrxn Nice work! I was able to install that dev version locally and get it working. I can see all the expected files have been included here:

https://unpkg.com/browse/@sketchbook-js/sketchbook@dev/

It's a prerelease for v0.2 instead of v0.1, but that's not problem. We'd need to more careful if there are people using it, but at this stage it's not a problem — [any 0.x version is expected to be unstable](https://semver.org/#spec-item-4).

I've added one more item to that checklist to update the `CONTRIBUTING.md` file (also has references to the old repo).
